extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -132,6 +132,35 @@ class MultipleBackendTask extends ReplicateObject {
         });
     }
 
+    /**
+     * Used when an operation performed in the process of replicating a
+     * multipart upload fails.
+     * @param {ObjectQueueEntry} sourceEntry - The source object entry
+     * @param {String} uploadId - The upload ID of the initiated MPU
+     * @param {Werelogs} log - The logger instance
+     * @param {Function} cb - The callback to call
+     * @return {undefined}
+     */
+    _handleMPUFailure(sourceEntry, uploadId, log, cb) {
+        this._multipleBackendAbortMPU(sourceEntry, uploadId, log, err => {
+            if (err) {
+                log.error('an error occurred when aborting a MPU', {
+                    method: 'MultipleBackendTask._handleMPUFailure',
+                    entry: sourceEntry.getLogInfo(),
+                    origin: 'target',
+                    peer: this.destBackbeatHost,
+                    error: err.message,
+                });
+                return cb(err);
```

Comment:
You could do `return cb()` since the error is not meant to be reused.

Reply:
I decided to pass it here even though I'm not using it because maybe someday, someone, somewhere, will expect an error from this method (as is the common callback pattern). I figured it best to allow the caller of the method to decide whether or not to use the error.